ci/Makefile

```diff
@@ -40,41 +36,12 @@ restart-proxy:
 	  --dry-run) > proxy.log 2>proxy.err & echo $$! > proxy.pid
 	sleep 2 && kill -0 $$(cat proxy.pid)
 
-restart-batch-proxy:
-	-kill $(shell cat batch-proxy.pid)
-	-kill -9 $(shell cat batch-proxy.pid)
-	-rm -rf batch-proxy.pid
-	$(eval BATCH_POD := $(shell kubectl get pods \
-                           -l app=batch \
-                           --field-selector=status.phase==Running \
-                           -o name \
-                         | sed 's:pods/::' \
-                         | head -n 1))
-	kubectl port-forward ${BATCH_POD} 8888:5000 > batch-proxy.log 2>batch-proxy.err & echo $$! > batch-proxy.pid
-	sleep 2 && kill -0 $$(cat batch-proxy.pid)
-
-
-
-run-local: HAIL_CI_REMOTE_PORT = 3000
-run-local: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
-run-local: restart-all-proxies
-	SELF_HOSTNAME=http://${PROXY_IP}:${HAIL_CI_REMOTE_PORT} \
-	BATCH_SERVER_URL=http://127.0.0.1:${HAIL_CI_LOCAL_BATCH_PORT} \
-	python3 run_ci.py
-
-run-local-for-tests: HAIL_CI_REMOTE_PORT = 3001
-run-local-for-tests: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
-run-local-for-tests: restart-all-proxies
-	SELF_HOSTNAME=http://${PROXY_IP}:${HAIL_CI_REMOTE_PORT} \
-	BATCH_SERVER_URL=http://127.0.0.1:${HAIL_CI_LOCAL_BATCH_PORT} \
-	WATCHED_TARGETS='[["hail-is/ci-test:master", true]]' \
-	python3 -m pip install --user ../batch && python3 run_ci.py
-
 test-locally: HAIL_CI_REMOTE_PORT = 3001
 test-locally: PROXY_IP=$(shell gcloud compute instances list --filter=name=dk-test --format="value(EXTERNAL_IP)")
-test-locally: restart-all-proxies
+test-locally: restart-proxy
```

Comment:
I think you need the make targets from `../cloud-sql.mk` to install cloud_sql_proxy and get the secret keys in the right place.

Reply:
good catch, I've added dependencies and also moved the PHONY targets to all be in `cloud-sql.mk`